packages/core/src/base/icons.scss

```diff
@@ -0,0 +1,208 @@
+/* stylelint-disable selector-class-pattern */
+@import '@cmsgov/design-system-support/src/settings/index';
+
+/*
+Icons
+
+Iconography should be used sparingly and only with support of labels. We use a subset of FontAwesome icons, with one or two bespoke icons.
+
+Please refer to [USWDS' documentation](https://designsystem.digital.gov/components/icons/) for more detail.
+
+Style guide: style.icons
+*/
+
+/*
+Alert Icons
+
+@hide-markup
+
+These icons are used primarily as part of the Alert Box component, though the Checkmark can sometimes be found in other places, without the background.
+
+Markup:
+<% var icons = [
+  'information', 'success', 'success-fill', 'warning', 'error'
+]; -%>
+<div class="ds-l-container">
```

Comment:
Wondering about changing this class from `ds-l-container` to `ds-u-padding--1` as it pulls the icons a bit closer to the left edge. 
### ds-u-padding--1 screenshot
![Screen Shot 2019-07-09 at 10 03 53 AM](https://user-images.githubusercontent.com/1449852/60894540-e96ee680-a230-11e9-98a1-4f665c3718c0.png)

### ds-l-container screenshot
![Screen Shot 2019-07-09 at 10 03 59 AM](https://user-images.githubusercontent.com/1449852/60894576-f5f33f00-a230-11e9-8b4d-929d9d19b424.png)

Reply:
sure why not